pages/api/sequences.py

```diff
@@ -432,3 +435,47 @@ def perform_destroy(self, instance):
                 self.sequence.has_certificate = False
                 self.sequence.save()
             instance.delete()
+
+
+class LiveEventListCreateAPIView(AccountMixin, PageElementMixin, ListCreateAPIView):
+    '''
+    Lists Live Events belonging to a Page Element
+    '''
+    serializer_class = LiveEventSerializer
+
+    def get_queryset(self):
+        queryset = LiveEvent.objects.filter(
+            element=self.element
+            ).order_by('-status', 'rank')
+        # if self.element_url_kwarg in self.kwargs:
+        #     queryset = queryset.filter(element=self.element)
+        return queryset
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+    def perform_create(self, serializer):
+        serializer.save(element=self.element, status='scheduled')
+
+
+class LiveEventRetrieveUpdateDestroyAPIView(PageElementMixin, RetrieveUpdateDestroyAPIView):
+    '''
+    Retrieve a LiveEvent
+    '''
+    serializer_class = LiveEventSerializer
+    # Doesn't allow editing the Status field because it is a read_only
+    # field in the Serializer since we're using the Delete method
+    # to set it.
+
+    def get_object(self):
+        return get_object_or_404(LiveEvent.objects.all(),
+            element=self.element, rank=self.kwargs.get('rank'))
+
+    def delete(self, request, *args, **kwargs):
+        live_event = self.get_object()
+        live_event.status = 'cancelled'
```

Comment:
`live_event.status = LiveEvent.CANCELED` and define `CANCELED` in `LiveEvent` model class as `CANCELED = 'canceled'.